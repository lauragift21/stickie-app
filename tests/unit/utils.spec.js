import { render, screen } from '@testing-library/vue'
import { randomColorGenerator } from '../../src/utils/index.js'

describe('generate random color', () => {
  it('should return a snapshot', () => {
    const color = randomColorGenerator()
    expect(color).toMatchSnapshot()
  })
})
