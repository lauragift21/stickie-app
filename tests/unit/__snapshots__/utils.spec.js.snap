// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generate random color should return a snapshot 1`] = `"#914EC3"`;

exports[`test random color generator function should return a color 1`] = `"#99DA39"`;

exports[`test random color generator function should return a snapshot 1`] = `"#930D10"`;
